warehouse/static/sass/blocks/_checkbox-tree.scss

```diff
@@ -39,3 +43,41 @@
     line-height: 1;
   }
 }
+
+
+.markup {
+
+  ul.tree {
+    list-style: none;
+    margin: 0;
+    padding: 0;
+   }
+
+  ul.tree ul {
+    list-style: none;
+    margin: 0;
+    padding: 0;
+   }
+
+  ul.tree ul {
+    margin-left: 10px;
+   }
+
+  ul.tree li {
+    margin: 0;
+    padding: 3px 7px;
+    line-height: 20px;
+    color: #464646;
```

Comment:
this particular color is part of the [predefined colors](https://github.com/pypa/warehouse/blob/master/warehouse/static/sass/settings/_colours.scss#L30) you should be able to just use `color: $text-color;` here